src/styles/style.scss

```diff
@@ -365,6 +365,34 @@ main {
   pointer-events: none;
   color: white;
 }
+.path.dark-gray-legend {
```

Comment:
Will be nice to implement all popover style inside `styles/molecules/popover.scss` (thus file doesn't exist yet) and you can put there standard style for popovers, together with extra styling for legend. :+1: